### ISSUE TITLE
Should we test against different Django/Wagtail versions?

### ISSUE THREAD

**Issue by loicteixeira** [springload/wagtaildraftail#22]
TL;DR: Should we test against different Django/Wagtail versions in addition to different Python versions?

52176e1935a416cb4ff4b1c12b526f6fd7e3c560 add a fix for `Django >= 1.8` so it would make sense for this package to be tested against different versions of Django (and Wagtail eventually).

Looking at what Wagtail does ([.travis.yml](https://github.com/wagtail/wagtail/blob/a2d7b133fcf59fc105b7fafbf552a15b2f113303/.travis.yml) and [tox.ini](https://github.com/wagtail/wagtail/blob/a2d7b133fcf59fc105b7fafbf552a15b2f113303/tox.ini)), they  only test for Django 1.8 with Python 2.7 and test with Django 1.10 and 1.11 on all other Python versions. I imagine this is because the those are the version they have branches for in the code (just like our commit above), although it's not clear why they don't test 1.10 with Python 2.7 given it is officially supported by Django.

Regarding Wagtail, the test app has been build for 1.9 and looking at the code, should work with previous versions, although we might want to be careful.

FWIW, I'm experimenting with such config on `chore/ci-config`.

**Comment by thibaudcolas** [comment]
Good question. My gut feeling is that multiple Django versions are worth supporting, but Wagtail versions aren't. That said, I don't know how easy it is to update Wagtail and how frequently users do it.

**Comment by dnsl48** [comment]
I believe we should test against different versions of both Django and Wagtail, since we depend on both Django models and Wagtail blocks.
